xilem_core/src/any_view.rs

```diff
@@ -206,20 +212,23 @@ where
         &self,
         view_state: &mut Self::ViewState,
         id_path: &[crate::ViewId],
-        message: crate::DynMessage,
+        message: Message,
         app_state: &mut State,
-    ) -> crate::MessageResult<Action> {
+    ) -> crate::MessageResult<Action, Message> {
         self.dyn_message(view_state, id_path, message, app_state)
     }
 }
 
 // TODO: IWBN if we could avoid this
```

Comment:
I think you mean the duplication of all the variations with `Send` and `Sync` for `dyn AnyView`. A different way to do this would be with our beloved macros I guess, but I think it's the right call to not do this with 4 repetitions...

Reply:
Ah yeah, right. That makes sense. It's not clear without the context of the nearby code. I think it's *probably* fine as-is